commit ef00d338ce99daa5cb321c5ac866a440f55fb572
Author: root <root@took23.ics.muni.cz>
Date:   Wed Oct 5 14:59:48 2016 +0200

    meta-defaultcfg.patch
    preservefqdn, stats, ratelimit
    117780: perian - kernel vypisuje chybove hlasky do konzole

diff --git a/debian/rsyslog.conf b/debian/rsyslog.conf
index 607b302..92cb573 100644
--- a/debian/rsyslog.conf
+++ b/debian/rsyslog.conf
@@ -3,6 +3,7 @@
 #			For more information see
 #			/usr/share/doc/rsyslog-doc/html/rsyslog_conf.html
 
+$PreserveFQDN on
 
 #################
 #### MODULES ####
@@ -10,7 +11,16 @@
 
 module(load="imuxsock") # provides support for local system logging
 module(load="imklog")   # provides kernel logging support
-#module(load="immark")  # provides --MARK-- message capability
+module(load="immark")  # provides --MARK-- message capability
+
+$ModLoad impstats
+$PStatInterval 600
+$PStatSeverity 7
+$PStatJSON on
+ 
+$IMUXSockRateLimitInterval 3600
+$IMUXSockRateLimitBurst 100000
+$RepeatedMsgReduction on
 
 # provides UDP syslog reception
 #module(load="imudp")
@@ -96,4 +106,4 @@ news.notice			-/var/log/news/news.notice
 #
 # Emergencies are sent to everybody logged in.
 #
-*.emerg				:omusrmsg:*
+*.emerg				:omusrmsg:root
